<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibraterView</class>
 <widget class="QWidget" name="CalibraterView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>606</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>标定</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2">
    <widget class="QListWidget" name="listWidget">
     <property name="maximumSize">
      <size>
       <width>240</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(129, 129, 129);</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGraphicsView" name="graphicsView">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(181, 181, 181);</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>标定设置</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QCheckBox" name="checkBox_CALIB_FIX_PRINCIPAL_POINT">
           <property name="text">
            <string>不优化主点位置</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" rowspan="5">
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QCheckBox" name="checkBox_CALIB_RATIONAL_MODEL">
           <property name="text">
            <string>使用理想模型(8参数)</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3" rowspan="5">
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QCheckBox" name="checkBox_CALIB_TILTED_MODEL">
           <property name="text">
            <string>使用倾斜传感器模型(14参数)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBox_CALIB_FIX_ASPECT_RATIO">
           <property name="text">
            <string>固定宽高比</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="checkBox_CALIB_THIN_PRISM_MODEL">
           <property name="text">
            <string>使用薄棱镜模型(12参数)</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <widget class="QCheckBox" name="checkBox_CALIB_FIX_TAUX_TAUY">
           <property name="text">
            <string>忽略倾斜传感器参数TauX和TauY</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBox_CALIB_ZERO_TANGENT_DIST">
           <property name="text">
            <string>忽略切向畸变系数(P1,P2)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QCheckBox" name="checkBox_CALIB_FIX_S1_S2_S3_S4">
           <property name="text">
            <string>忽略薄棱镜参数S1-S4</string>
           </property>
          </widget>
         </item>
         <item row="2" column="4">
          <widget class="QCheckBox" name="checkBox_CALIB_SAME_FOCAL_LENGTH">
           <property name="text">
            <string>强制两侧相机焦距相等</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBox_CALIB_FIX_K1">
           <property name="text">
            <string>忽略径向畸变系数K1</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QCheckBox" name="checkBox_CALIB_FIX_K3">
           <property name="text">
            <string>忽略径向畸变系数K3</string>
           </property>
          </widget>
         </item>
         <item row="3" column="4">
          <widget class="QCheckBox" name="checkBox_CALIB_FIX_K5">
           <property name="text">
            <string>忽略径向畸变系数K5</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QCheckBox" name="checkBox_CALIB_FIX_K2">
           <property name="text">
            <string>忽略径向畸变系数K2</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QCheckBox" name="checkBox_CALIB_FIX_K4">
           <property name="text">
            <string>忽略径向畸变系数K4</string>
           </property>
          </widget>
         </item>
         <item row="4" column="4">
          <widget class="QCheckBox" name="checkBox_CALIB_FIX_K6">
           <property name="text">
            <string>忽略径向畸变系数K6</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>标定参数</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Width:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_BoardWidth">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Height:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_BoardHeight">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Size:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_SquareSize">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>20</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>20</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string>标定</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>4</number>
         </property>
         <property name="leftMargin">
          <number>4</number>
         </property>
         <property name="topMargin">
          <number>4</number>
         </property>
         <property name="rightMargin">
          <number>4</number>
         </property>
         <property name="bottomMargin">
          <number>4</number>
         </property>
         <item>
          <widget class="QPushButton" name="pushButton_CalibDefault">
           <property name="text">
            <string>默认参数</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton">
           <property name="text">
            <string>标定</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_2">
           <property name="text">
            <string>保存参数</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit_BoardWidth</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CalibraterView</receiver>
   <slot>onValueChanged_BoardWidth(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>329</y>
    </hint>
    <hint type="destinationlabel">
     <x>246</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_BoardHeight</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CalibraterView</receiver>
   <slot>onValueChanged_BoardHeight(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>141</x>
     <y>365</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_SquareSize</sender>
   <signal>textChanged(QString)</signal>
   <receiver>CalibraterView</receiver>
   <slot>onValueChanged_SquareSize(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>396</y>
    </hint>
    <hint type="destinationlabel">
     <x>174</x>
     <y>283</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_CalibDefault</sender>
   <signal>clicked()</signal>
   <receiver>CalibraterView</receiver>
   <slot>setDefaultCalibParam()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>455</x>
     <y>438</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>472</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>CalibraterView</receiver>
   <slot>doCalib()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>542</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>569</x>
     <y>469</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onValueChanged_BoardWidth(QString)</slot>
  <slot>onValueChanged_BoardHeight(QString)</slot>
  <slot>onValueChanged_SquareSize(QString)</slot>
  <slot>onClicked_showRectified(bool)</slot>
  <slot>onClicked_Bouguet(bool)</slot>
  <slot>onClicked_Hartley(bool)</slot>
  <slot>setDefaultCalibParam()</slot>
  <slot>doCalib()</slot>
 </slots>
</ui>
